sklearn/metrics/pairwise.py

```diff
@@ -203,7 +203,7 @@ def euclidean_distances(X, Y=None, Y_norm_squared=None, squared=False,
 
     Returns
     -------
-    distances : {array, sparse matrix}, shape (n_samples_1, n_samples_2)
+    distances : array, shape (n_samples_1, n_samples_2)
```

Comment:
I put the comment here but I think that it should be written above.

I would either add a note (using sphinx syntax) or within the parameter description that `X_norm_squared` and `Y_norm_squared` will be ignored if they are passed as `float32` since they will lead to inaccurate distances.